### ISSUE TITLE
custom api need more explanations 

### ISSUE THREAD

**Issue by onemen** [neutralinojs/neutralinojs.github.io#256]
The description is [custom.md](https://github.com/neutralinojs/neutralinojs.github.io/edit/main/docs/api/custom.md) is cryptic. 
```
inbuilt helper functions (i.e., to get the window handler,
validation, etc.), and a developer guide to add custom APIs to the Neutralinojs framework without
updating the framework core.
```
`inbuilt helper functions` ???

Where is the **developer guide to add custom APIs to the Neutralinojs framework without
updating the framework core**

**Comment by RitikRG** [comment]
Hey there!
I am willing to work on this issue, can you assign me this issue.

**Comment by shalithasuranga** [comment]
Hey @onemen, thanks for reporting the issue. Seems like we can extend the documentation with [this](https://github.com/neutralinojs/neutralinojs/blob/c908c628fc5aa1ad3f445202aad452dce342ba84/api/custom/custom.cpp#L44) example by explaining more about the inbuilt helper functions, etc.. @RitikRG sure, please submit a pull request if would like to contribute.. 

Thanks :tada:

**Comment by RitikRG** [comment]
@shalithasuranga thanks for the opportunity 
Just to confirm, you want me to make changes to the custom.md--> custom.getMethods() section file based on [this](https://github.com/neutralinojs/neutralinojs/blob/c908c628fc5aa1ad3f445202aad452dce342ba84/api/custom/custom.cpp#L44) 

I am new to open source, so that's why I am a little confused

**Comment by RitikRG** [comment]
Hey I have made a pr #260 , please check it out
Thank you 🎉

**Comment by RitikRG** [comment]
Hey @shalithasuranga ,
Sorry to interrupt, is there anything that is to be done to this pr.
Thank you

**Comment by skushagra9** [comment]
Hey, I am working on this

**Comment by shalithasuranga** [comment]
Closing this since the issue was fixed via #298. Thanks all :tada:
